blaze/mahimahi/server/filestore.py

```diff
@@ -68,21 +99,20 @@ def read(path: str) -> "File":
         # Decode headers from a list of pairs to a dictionary, decoding bytes to str and converting to lowercase
         # to make easier parsing. Also remove headers that we don't want to send back in a replayed response
         req_headers = {h.key.decode().lower(): h.value.decode() for h in record.request.header}
-        res_headers = {
-            h.key.decode().lower(): h.value.decode()
-            for h in record.response.header
-            if h.key.decode().lower() not in REMOVE_HEADERS
-        }
+        res_headers = {h.key.decode().lower(): h.value.decode() for h in record.response.header}
 
-        body = record.response.body
+        # Pushable objects must be cacheable
+        is_cacheable = check_cacheability(res_headers)
 
         # Unchunk the body if it is chunked since HTTP/2 does not support chunked encoding
+        body = record.response.body
         if TRANSFER_ENCODING_HEADER in res_headers and "chunked" in res_headers[TRANSFER_ENCODING_HEADER].lower():
```

Comment:
on the right side, if you unchunk the body, should we delete the transfer_encoding _header like in the old version?

Reply:
The header has been added to `REMOVE_HEADERS` and will be removed on line 113